docs/primitive-types.md

```diff
@@ -0,0 +1,55 @@
+---
+id: primitive-types
+title: Primitive Types
+sidebar_label: Primitive Types
+---
+
+In JavaScript, the **[primitive types](https://developer.mozilla.org/en-US/docs/Glossary/Primitive)** are types of values that aren't objects. This page gives some simple examples of how to construct instances of various JavaScript primitive types, and you can follow the links to more detailed information in the API documentation.
+
+## Numbers
+
+The [`Context::number()`](https://docs.rs/neon/latest/neon/context/trait.Context.html#method.number) method constructs a JavaScript number from any Rust number compatible with the [`f64`](https://doc.rust-lang.org/std/primitive.f64.html) type. This includes both integers and floating-point numbers, so you can conveniently construct a JavaScript number from a literal number:
+
+```rust
+let i: Handle<JsNumber> = cx.number(42);
+let f: Handle<JsNumber> = cx.number(3.14);
+```
+
+For types that aren't implicitly convertible to `f64`, you can explicitly cast a number with Rust's [`as`](https://doc.rust-lang.org/std/keyword.as.html) operator:
```

Comment:
It might be worth mentioning that this can silently cause loss of precision and invalid results. `TryFrom` or (some crate) may be preferred.